src/overlays/actors/ovl_Bg_Breakwall/z_bg_breakwall.c

```diff
@@ -10,18 +10,31 @@
 
 #define THIS ((BgBreakwall*)thisx)
 
+typedef struct {
+    /* 0x00 */ CollisionHeader* colHeader;
+    /* 0x04 */ Gfx* dList;
+    /* 0x08 */ s8 colType;
+} BombableWallInfo;
+
+typedef enum {
+    /* 0 */ BWALL_DC_ENTRANCE,  // When exploded it will play the Dodongo's Cavern intro cutscene
+    /* 1 */ BWALL_WALL,         // Used a lot in Dodongo's Cavern and other places
+    /* 2 */ BWALL_KD_FLOOR,     // Used in the King Dodongo boss room
+    /* 3 */ BWALL_KD_LAVA_COVER // Spawned by King Dodongo to cover the floor of the boss room after the fight so the
+                                // lava does not hurt the player anymore
+} BombableWallType;
```

Comment:
this needs to be in the header, since it will be accessed externally.
I would also try to condense the lava floor comment to a single line if possible